### PR TITLE
Integrate OCEX pallet to runtime 

### LINKED ISSUE #433: Integrate OCEX pallet to mainnet
Integrate the ocex pallet to the runtime of mainnet

### PR DESCRIPTION
**Pull request by felixfaisal**
This PR adds and integrates the OCEX pallet to the runtime

### CODE REVIEW

File: pallets/ocex/src/benchmarking.rs [hunk 1]
```diff
@@ -0,0 +1,78 @@
+// This file is part of Substrate.
```
Comment (zktony) [inline pallets/ocex/src/benchmarking.rs]: Please update the copyright.

File: pallets/ocex/src/benchmarking.rs [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+use crate::*;
+use frame_benchmarking::{benchmarks, whitelisted_caller};
+use frame_system::RawOrigin;
+
+// To actually run this benchmark on pallet-example-basic, we need to put this pallet into the
+//   runtime and compile it with `runtime-benchmarks` feature. The detail procedures are
+//   documented at:
+//   https://docs.substrate.io/v3/runtime/benchmarking#how-to-benchmark
+//
```
Comment (zktony) [inline pallets/ocex/src/benchmarking.rs]: please remove these comments

File: pallets/ocex/src/benchmarking.rs [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+		for i in (0..x).rev() {
+			m.push(i);
+		}
+	}: {
+		// The benchmark execution phase could also be a closure with custom code
+		m.sort();
+	}
+
+	// This line generates test cases for benchmarking, and could be run by:
+	//   `cargo test -p pallet-example-basic --all-features`, you will see one line per case:
```
Comment (zktony) [inline pallets/ocex/src/benchmarking.rs]: same

File: pallets/ocex/src/benchmarking.rs [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+//   https://docs.substrate.io/v3/runtime/benchmarking#how-to-benchmark
+//
+// The auto-generated weight estimate of this pallet is copied over to the `weights.rs` file.
+// The exact command of how the estimate generated is printed at the top of the file.
+
+// Details on using the benchmarks macro can be seen at:
+//   https://paritytech.github.io/substrate/master/frame_benchmarking/trait.Benchmarking.html#tymethod.benchmarks
+benchmarks! {
+	// This will measure the execution time of `set_dummy` for b in [1..1000] range.
+	set_dummy_benchmark {
```
Comment (zktony) [inline pallets/ocex/src/benchmarking.rs:25]: These benchmarks are for example pallet.

File: pallets/ocex/src/tests.rs [hunk 5]
```diff
@@ -0,0 +1,134 @@
+// This file is part of Substrate.
+
+// Copyright (C) 2019-2022 Parity Technologies (UK) Ltd.
```
Comment (zktony) [inline pallets/ocex/src/tests.rs]: Copyright

**Review by zktony — CHANGES_REQUESTED** [review]

File: pallets/ocex/src/benchmarking.rs (hunk 4 above)
Comment (zktony) [reply pallets/ocex/src/benchmarking.rs:25]: will be handled in #435

**Review by zktony — APPROVED** [review]

**Review by Gauthamastro — APPROVED** [review]

**Pull request merged**